gap/fp/word.gi

```diff
@@ -0,0 +1,14 @@
+InstallMethod(WordToString, "for a string and a list",
+[IsString, IsList],
+function(alphabet, word)
+  local str, i, j;
+   str := "";
+   j := PositionMaximum(word);
+   if word[j] > Length(alphabet) then
+     return "Error, there are not enough letters in the alphabet";
```

Comment:
This isn't the correct way to do an error in GAP, you should use `ErrorNoReturn`, please look at another file in Semigroups to see an example.